2024-08-09 06:42:14,179 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.18:43890'
2024-08-09 06:42:14,183 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.18:41273'
2024-08-09 06:42:14,775 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-wynzjaac', purging
2024-08-09 06:42:14,776 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-uyz9iu7y', purging
2024-08-09 06:42:15,286 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.18:42261
2024-08-09 06:42:15,286 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.18:44764
2024-08-09 06:42:15,287 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.18:42261
2024-08-09 06:42:15,287 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.18:44764
2024-08-09 06:42:15,287 - distributed.worker - INFO -           Worker name:           SLURMCluster-0-1
2024-08-09 06:42:15,287 - distributed.worker - INFO -           Worker name:           SLURMCluster-0-0
2024-08-09 06:42:15,287 - distributed.worker - INFO -          dashboard at:          10.34.59.18:36984
2024-08-09 06:42:15,287 - distributed.worker - INFO -          dashboard at:          10.34.59.18:38398
2024-08-09 06:42:15,287 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:36549
2024-08-09 06:42:15,287 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:36549
2024-08-09 06:42:15,287 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,287 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,287 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:42:15,287 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:42:15,287 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:42:15,287 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:42:15,287 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-r81eht1h
2024-08-09 06:42:15,287 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-33sztfal
2024-08-09 06:42:15,287 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,287 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,691 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:42:15,691 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:42:15,691 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:36549
2024-08-09 06:42:15,691 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,692 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:36549
2024-08-09 06:42:15,692 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:36549
2024-08-09 06:42:15,692 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:42:15,692 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:36549
slurmstepd: error: *** JOB 2330893 ON cl-node018 CANCELLED AT 2024-08-09T06:42:25 ***
2024-08-09 06:42:25,431 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.18:42261. Reason: scheduler-close
2024-08-09 06:42:25,431 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.18:44764. Reason: scheduler-close
2024-08-09 06:42:25,432 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.18:57027 remote=tcp://10.34.59.1:36549>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.18:57027 remote=tcp://10.34.59.1:36549>: Stream is closed
2024-08-09 06:42:25,433 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.18:57026 remote=tcp://10.34.59.1:36549>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.18:57026 remote=tcp://10.34.59.1:36549>: Stream is closed
2024-08-09 06:42:25,435 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.18:41273'. Reason: scheduler-close
2024-08-09 06:42:25,436 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.18:43890'. Reason: scheduler-close
2024-08-09 06:42:25,438 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:36549; closing.
2024-08-09 06:42:25,438 - distributed.nanny - INFO - Worker closed
2024-08-09 06:42:25,439 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:36549; closing.
2024-08-09 06:42:25,439 - distributed.nanny - INFO - Worker closed
